docs/source/deployment/now.md

```diff
@@ -25,33 +30,50 @@ Deployment to Now for Node.js apps simply requires a `package.json` file to be p
 }
 ```
 
-### Deploy server to Now
+Now looks for a `start` script to start the app. As long as that is in the `package.json`, the app should be able to start up.
+
+## Deploying with Now
 
-Install the [now CLI](https://zeit.co/download#now-cli), then visit your server directory and run the `now` command:
+Now offers multiple options for deploying projects.
+
+### Local Projects
+
+If the Now CLI is installed, then running the `now` command from the root directory of the project should deploy the project.
 
 ```sh
 $ now
 ```
 
-The `now` command immediately deploys your server to the cloud and returns the hosted project link. Send a query to your GraphQL server on `now` at `<NOW_APP_NAME>.now.sh`.
+The `now` command immediately deploys a server to the cloud and returns the hosted project link. After finishing, it should be possible to send a query to the GraphQL schema on `now` at `<NOW_APP_NAME>.now.sh/graphql`.
 
-### Deploying directly from GitHub
+### Manual GitHub deployment
 
-If you have your GraphQL server published to GitHub, Now provides the ability to deploy straight from GitHub to the cloud.
+If a GraphQL server project is publically available on GitHub, Now provides the ability to deploy straight from GitHub by calling the `now` command with the user/organization name and repository name in the format `user/repository-name`.
 
-Assuming you'd like to deploy an instance of [apollo](https://github.com/apollographql)'s [graphql-server-example](https://github.com/apollographql/graphql-server-example), this is what you'll do:
+For example, to deploy [Apollo](https://github.com/apollographql)'s [graphql-server-example](https://github.com/apollographql/graphql-server-example), the command would be:
 
 ```sh
 $ now apollographql/graphql-server-example
 ```
 
-The `now` command deploys right away and gets the server up and running on the cloud. Furthermore, running the following command will automatically start delivering reports to Apollo Engine.
+The `now` command deploys right away and attempts to start the server This specific example would fail though, due to missing environment variables. There's more info on that in the [next section](#env-variables).
+
+<h3 id="automatic-github-deploys">Automatic GitHub deployment</h3>
+
+Now supports automatic deployments from GitHub on pull requests.
+
+To set up automatic deplotments, visit [https://zeit.co/github](https://zeit.co/github) and click on the "Setup Now" button.
+
+After signing in with GitHub, a GitHub app can be added to any account or organization. Once installed, it's possible to choose which Repositories that Now will run on, and will create new deployments on Pull Requests.
```

Comment:
I'm not sure where `a GitHub app` comes from. Maybe `the Now GitHub app can ...`

Reply:
The tense is a bit off here and capitalization of repositories is probably not necessary. Also, I'm not sure what installation means here. Maybe activated for an org/account or does it mean activation for a repository.